### ISSUE TITLE
Can you please share example input files and output results generated?

### ISSUE THREAD

**Issue by SandeepManjanna** [Sahil-Ajmera/Orienteering-problem-solution-using-A-star-and-bfs#1]
Can you please share example input files and output results generated? This would be useful to use the code for some quick tests.

**Comment by Sahil-Ajmera** [comment]
Hi ! Thanks for reaching out.  I apologize for not having added them in the first place.

Inputs:
map.png - Input map
mpp.txt - elevation values
red.txt - sample points
  each line is a set of points on the map
first line is a start point
Each subsequent line is a control point
last line is a end point.

Output:
See terrain-fall file for change of terrain during fall
See terrain-falloutput file for the result during that season
 
Basically we need to draw the path on the map using the start point , set of control points and the end point.

Hope this helps. :)

**Comment by scientific-sampling-robots** [comment]
Hi Sahil,
Thanks for your reply. Can you comment how long it takes for the code to run with the standard input files? I have been running it from past 15 hours and it seems to be still running.

Thanks,
Sandeep

**Comment by Sahil-Ajmera** [comment]
Hi ! It shouldn't be taking that long at all. I remember it took only a minute or two in the worst case. Are you sure you used the right input files ?
Here I tried running the same code with the inputs presented here. terrain.png is nothing but map.png
The output would be a png file with the actual orienteering done on the input map and when the season is fall.
![image](https://user-images.githubusercontent.com/32546795/67643829-f6a00600-f8f1-11e9-917b-2afe56c32107.png)
Input terrain.png / map.png ( in this repo )
![image](https://user-images.githubusercontent.com/32546795/67643882-60201480-f8f2-11e9-8fd3-2b888c9d057d.png)

Output
![image](https://user-images.githubusercontent.com/32546795/67643869-4bdc1780-f8f2-11e9-8488-f90b271d7c3d.png)
